client/src/components/Map.js

```diff
@@ -33,25 +33,37 @@ class Map extends Component {
     const canvas = this.refs.canvas
     const ctx = canvas.getContext('2d')
 
-    let gameStateReq = await window.fetch('http://localhost:3006/zones')
+    let gameStateReq = await window.fetch('http://localhost:3006/gameStates/0')
+    let zoneReq = await window.fetch('http://localhost:3006/zones/0')
+    let factionReq = await window.fetch('http://localhost:3006/factions/0')
```

Comment:
Consider use of [`react-refetch`](https://github.com/heroku/react-refetch) and pass this data in via props rather than having a stateful component?